docs/how-to-guides/adding-a-new-offline-store.md

```diff
@@ -260,23 +264,56 @@ driver_hourly_stats_view = FeatureView(
 
 ## 6. Testing the OfflineStore class
 
+### Contrib
```

Comment:
This doesn't seem like it logically belongs under "testing" ?

Reply:
Moved to `Defining an OfflineStore class`